lib/watcher-middleware.js

```diff
@@ -34,6 +36,23 @@ module.exports = function watcherMiddleware(watcher, options) {
 
       next();
     }, (buildError) => {
+      if (request.url === '/ember_open_editor_for_template_compilation_error') {
+        let errorLocation = buildError.broccoliPayload.error.location || {};
+        let fileName = errorLocation.file;
+        if (fileName) {
+          let normalisedFileName = normaliseFileName(fileName);
```

Comment:
I think we should add a check here whether the file actually exists. Or the `normaliseFileName` function could be aware of the filesystem and try `fs.existsSync()` checks on various paths until it finds a matching one that exists and otherwise return `null`.